TypeCobol/Compiler/Text/CobolChar.cs

```diff
@@ -88,6 +105,30 @@ static CobolChar()
             isCobolWordSeparator[(byte)'='] = true;
             isCobolWordSeparator[(byte)'"'] = true;
             isCobolWordSeparator[(byte)'\''] = true;
+
+            // --- isAdditionalCharAllowedInPartialCobolWord ---
+
+            for (int i = 0; i < 256; i++)
+            {
+                isAdditionalCharAllowedInPartialCobolWord[i] = false;
+            }
```

Comment:
I think we should remove those unneeded inits in this file (and add a comment stating that bool default value is `false`)

Reply:
Done